core/nodejsActionBase/app.js

```diff
@@ -38,7 +38,7 @@ app.use(bodyParser.json({ limit: "48mb" }));
 app.post('/init', wrapEndpoint(service.initCode));
 app.post('/run',  wrapEndpoint(service.runCode));
 
-app.use(function(err, req, res, next) {
+app.use(function(req, res, next) {
     console.error(err.stack);
```

Comment:
is err now undefined on line 42? 
did this middleware interface change?

Reply:
i see in your description you intended to remove line 42 perhaps?

Reply:
Could not find docs that supported that interface for 4.x 
https://expressjs.com/en/guide/using-middleware.html

Reply:
even in deprecated 3.x: it was not in the framework docs:
https://expressjs.com/en/3x/api.html#middleware

Reply:
I did find what likely someone copied verbatim for creating your own error handler here:https://expressjs.com/en/guide/error-handling.html
but, in standalone testing the function was never called...

Reply:
Anyone able to point to any tests that exhibit the existing impl.?

Reply:
i think it's ok to remove this.

Reply:
If I hit the existing code with an POST/GET (etc.) to a bogus endpoint, I get a default 404 error:
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>Error</title>
</head>
<body>
<pre>Cannot POST /bad</pre>
</body>
</html>


Response code: 404 (Not Found); Time: 68ms; Content length: 143 bytes
```

if the intent is to immediately block all these bad requests immediately this change results in
```
HTTP/1.1 500 Internal Server Error
X-Powered-By: Express
Content-Type: application/json; charset=utf-8
Content-Length: 24
ETag: W/"18-928jOLijreS3ro9mYfVt0IU4k/0"
Date: Tue, 26 Feb 2019 20:03:54 GMT
Connection: keep-alive

{
  "error": "Bad request."
}

Response code: 500 (Internal Server Error); Time: 11928ms; Content length: 24 bytes
```

Trying to figure out the original intent and perhaps short circuit any endpoint that is not intended to be part of the actual runtime.